index.d.ts

```diff
@@ -5,6 +5,14 @@ declare module "gpt-3-encoder" {
 
     export function countTokens(text: string): number;
 
-    export function tokenStats(input: string | number[]): any;
+    export function tokenStats(input: string | number[]): TokenStats;
+
+    export interface TokenStats {
```

Comment:
Does this look right to you for ts syntax

Reply:
somone tested `npm install gptoken` and it worked in next.js with new types

Reply:
Yes, it is definitely works now!

Reply:
Note in the gptoken branch I also exposed some of the guts and utility functions in case they are useful.


This is the full type
```js

declare module "gptoken" {
    export function encode(text: string): number[];
    export function decode(tokens: number[]): string;
    export function countTokens(text: string): number;
    export function tokenStats(input: string | number[]): TokenStats;

    export interface TokenStats {
        count: number;
        unique: number;
        frequency: Record<string, number>;
        positions: Record<string, number[]>;
        tokens: string[];
    }
    export interface Utils {
        ord: (char: string) => number;
        chr: (code: number) => string;
        bpe: (codes: number[], options?: BpeOptions) => number[];
        range: (start: number, end: number, step?: number) => number[];
        pat: RegExp;
        get_pairs: (word: string) => string[];
        encodeStr: (str: string) => number[];
        decodeStr: (arr: number[]) => string;
    }

    export const util: Utils;

    export const maps: {
        encoder: Record<string, number>;
        decoder: Record<number, string>;
        byte_decoder: Record<number, string>;
        byte_encoder: Record<string, number>;
        bpe_ranks: Record<string, number>;
        cache: Map<string, number[]>;
    };
    export interface BpeOptions {
        isDict?: boolean;
        vocab?: Record<string, number>;
        merges?: Record<string, string>;
        separator?: string;
    }
}

```